engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -1,28 +1,29 @@
 package cromwell.engine.workflow
 
-import java.time.OffsetDateTime
-
 import akka.actor.FSM.{CurrentState, SubscribeTransitionCallBack, Transition}
 import akka.actor._
 import akka.event.Logging
 import akka.routing.FromConfig
 import com.typesafe.config.{Config, ConfigFactory}
-import cromwell.core.{WorkflowId, WorkflowSourceFiles}
-import cromwell.database.obj.WorkflowMetadataKeys
+import cromwell.core.WorkflowId
 import cromwell.engine._
 import cromwell.engine.workflow.WorkflowActor._
 import cromwell.engine.workflow.WorkflowManagerActor._
 import cromwell.engine.workflow.lifecycle.CopyWorkflowLogsActor
 import cromwell.services.MetadataServiceActor._
-import cromwell.services.{MetadataEvent, MetadataKey, MetadataValue, ServiceRegistryClient}
+import cromwell.services.ServiceRegistryClient
 import cromwell.webservice.CromwellApiHandler._
 import lenthall.config.ScalaConfig.EnhancedScalaConfig
 
 import scala.concurrent.duration._
 import scala.concurrent.{Await, Promise}
 import scala.language.postfixOps
+import scalaz.NonEmptyList
 
 object WorkflowManagerActor {
+  val DefaultMaxWorkflowsToRun = 5000
```

Comment:
I suppose that with them being configurable anyways it doesn't really matter. We can always fiddle with these if it becomes problematic for some reason